src/controllers/queries/index.ts

```diff
@@ -478,8 +478,13 @@ export class QueriesController extends HTMLController {
         await this.db.update('query', { id: query.id }, { query_response: localQuery.emissions, status: 'resolved' })
       }
 
+      const render =
+        method === 'submit_query_request'
+          ? this.scope3CarbonConsumptionResponseTemplates.newScope3CarbonConsumptionResponseFormPage
+          : this.scope3CarbonConsumptionTemplates.newScope3CarbonConsumptionFormPage
```

Comment:
is this fixing a bug? This feels like more than a copy change but that we're rendering something different? Also do either of these methods care about `this` binding?

Reply:
Indeed, from while was working on partials I did use incorrect newQuerySuccess one meant to be for requester and one for responder so I used the same, to be fair even text was the same (before the copy change)

Reply:
Ok so after discussion this does fix a bug. Because the methods are defined as `=>` functions `this` binding shouldn't be an issue

Reply:
agreed, updated to be just like: 

```js
      if (!parentId && localQuery.emissions) {
        await this.db.update('query', { id: query.id }, { query_response: localQuery.emissions, status: 'resolved' })
        return this.html(
          this.scope3CarbonConsumptionTemplates.newScope3CarbonConsumptionFormPage({
            formStage: 'success',
            company: {
              companyName: conn.company_name,
            },
          })
        )
      }
```